tests/cucumber/acceptance/pageObjects/Login.js

```diff
@@ -0,0 +1,21 @@
+const { config } = require("../../../../config");
+class Login {
+  constructor() {
+    this.loginPageUrl = config.hostUrl + "/login/index.php";
+    this.homePageUrl = config.hostUrl + "/my/";
+    this.username = config.adminUser;
+    this.password = config.adminPassword;
+    this.usernameSelector = "#username";
+    this.passwordSelector = "#password";
+    this.loginButton = "#loginbtn";
```

Comment:
if we follow above pattern then
```suggestion
    this.loginButtonSelector = "#loginbtn";
```